newIDE/app/src/UI/Accordion.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+import * as React from 'react';
+import MUIAccordion from '@material-ui/core/Accordion';
+import MUIAccordionSummary from '@material-ui/core/AccordionSummary';
+import MUIAccordionDetails from '@material-ui/core/AccordionDetails';
+import MUIAccordionActions from '@material-ui/core/AccordionActions';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { IconButton } from '@material-ui/core';
```

Comment:
nitpicking: use @material-ui/core/IconButton for consistency (and could help I think with code splitting in the future).

Reply:
Oh, I actually intended to use the `IconButton` implemented in newIDE, but I guess Flow auto-import picked the other one and I never noticed. I'll fix it.